march_state_machine/src/healthy_sm/moving_sm/moving_sm.py

```diff
@@ -1,16 +1,26 @@
 #!/usr/bin/env python
 import smach
 
-from Walking import Walking
+from states.PoseState import PoseState
 
 
 def create():
     sm_moving = smach.StateMachine(outcomes=['succeeded', 'failed'])
+
     # Open the container
     with sm_moving:
         # Add states to the container
-        smach.StateMachine.add('WALKING', Walking(),
+
+        smach.StateMachine.add('LeftSwingPose',
```

Comment:
```suggestion
        smach.StateMachine.add('LEFTSWINGPOSE',
```

StateNames preferred capitalized by SMACH conventions